packages/orchestration/src/chain-capabilities.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * @file Contains ChainInfo that not available from a well-known chain registry.
+ */
+
+/** @import {KnownChains} from '@agoric/orchestration'; */
+
+/**
+ * Chains with the async-icq (icq-1) module available.
+ *
+ * @satisfies {(keyof KnownChains)[]}
+ */
+export const ICQ_ENABLED = /** @type {const} } */ ['omniflixhub', 'osmosis'];
+harden(ICQ_ENABLED);
+
+/**
+ * Chains with the packet-forward-middleware module available.
+ *
+ * @satisfies {(keyof KnownChains)[]}
+ */
+export const PFM_ENABLED = /** @type {const} } */ [
```

Comment:
non-blocking: better to use an object map for faster lookup. also allows being explicit about chains where it's known to not be enabled (`: false`) whereas omission could mean either false or unknown.

ditto for ICQ_ENABLED above.

also I don't remember if our house style takes an opinion on casing these but it looks like a regular hardened object to me (i.e. `PfmEnabled`)